debian/kafka-connect-base/include/etc/confluent/docker/configure

```diff
@@ -54,6 +54,24 @@ then
   dub ensure CONNECT_VALUE_CONVERTER_SCHEMA_REGISTRY_URL
 fi
 
+if [ -z "${CONNECT_SSL_KEYSTORE_FILENAME+x}" ]
+then
+  dub ensure CONNECT_SSL_KEYSTORE_CREDENTIALS
+  CONNECT_SSL_KEYSTORE_CREDENTIALS_LOCATION="/etc/kafka/secrets/$CONNECT_SSL_KEYSTORE_CREDENTIALS"
```

Comment:
The mountpoint in the Dockerfile for secrets is`/etc/kafka-connect/secrets` so we should make it the same here too.